index.html

```diff
@@ -137,7 +137,15 @@ <h2 class="minitext">
   define("three", function() { return require("two") + require("one"); });
   define("four", function() { return require("three") + require("one"); });
 })();
-three = require("three");</code></pre>
+three = require("three");
```

Comment:
It aligns the scrollbar to the end of the code block, see before and after.
![2017-05-03-205103_3840x2160_scrot](https://cloud.githubusercontent.com/assets/4820517/25688567/de8488f6-3045-11e7-8bcb-0dcb2f7e409c.png)
![2017-05-03-205206_3840x2160_scrot](https://cloud.githubusercontent.com/assets/4820517/25688566/de77e3da-3045-11e7-9105-13a3dc036c4c.png)